packages/mina-entry-webpack-plugin/index.js

```diff
@@ -39,12 +39,23 @@ function getUrlsFromConfig(config) {
     urls = [...urls, ...config.pages]
   }
 
-  ;['pages', 'usingComponents', 'publicComponents'].forEach(prop => {
-    if (typeof config[prop] !== 'object') {
-      return
-    }
+  let components = ['pages', 'usingComponents', 'publicComponents'].map(
+    c => config[c]
+  )
+
+  if (Array.isArray(config['subPages'])) {
+    config['subPages'].forEach(subPage => {
+      const { root, pages } = subPage
+      if (Array.isArray(pages)) {
+        pages.forEach(page => {
```

Comment:
I think it should be ``components = [...components, pages.map(page => path.join(root, page))]``